src/GLideNUI/ConfigDialog.cpp

```diff
@@ -1113,7 +1116,7 @@ void ConfigDialog::on_btn_clicked() {
 			//msgBox.setIcon(QMessageBox::Information);
 			msgBox.exec();
 			if (msgBox.m_nativeVK != 0) {
-				const unsigned int hidCode = osal_virtual_key_to_hid(msgBox.m_nativeVK);
+				const unsigned int hidCode = osal_virtual_key_to_hid(msgBox.m_nativeVK, QtKeyToHID(msgBox.m_Key));
```

Comment:
QtKeyToHID makes osal_virtual_key_to_hid redundant. Lets remove it.

Reply:
there's a problem, i.e certain hotkeys wont work due to `Qt::Key` being limited (for example the keypad numkeys wont work on windows if I remove `osal_virtual_key_to_hid`), if you're fine with losing the ability to do such things, then sure I'll remove it.

Reply:
keypad numkeys aren't working on Windows with the current code, at least on my keyboard. If this is the only problem, please remove sal_virtual_key_to_hid.

Reply:
pushed